app/models/comment.rb

```diff
@@ -0,0 +1,5 @@
+class Comment < ApplicationRecord
+  belongs_to :commentable, polymorphic: true
+  belongs_to :user
+  validates  :body, presence: true, length: { maximum: 300 }
```

Comment:
commentable can be nil?

Reply:
@igorkasyanchuk here `commentable_type` may be anything user set in hidden input